CHANGELOG.md

```diff
@@ -6,7 +6,19 @@ This changelog follows [keepachangelog](https://keepachangelog.com/en/1.0.0/) fo
 This project versioning is _similar_ to [Semantic Versioning](https://semver.org), and is documented in [SEP 14](https://github.com/saltstack/salt-enhancement-proposals/pull/20/files).
 Versions are `MAJOR.PATCH`.
 
-## Unreleased (Neon)
+### Unreleased
```

Comment:
would this need to be `Unreleased (Sodium)` now?

Reply:
Yes, that's correct - good catch!

Reply:
My bad - that actually was supposed to be 3000.1 🤦‍♂